lib/src/voip/voip.dart

```diff
@@ -36,7 +36,13 @@ class VoIP {
   Map<VoipId, GroupCallSession> get groupCalls => _groupCalls;
   final Map<VoipId, GroupCallSession> _groupCalls = {};
 
-  final CachedStreamController<CallSession> onIncomingCall =
+  /// The stream is used to prepare for incoming peer calls in a mesh call
+  /// For example, registering listeners
```

Comment:
if these are just used in mesh calls, maybe `onIncomingCallInMeshStart` and `onIncomingCallInMeshSetup` would be better? but i'll leave that decision completely up to you